tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaAuthorizationTestBase.java

```diff
@@ -680,6 +680,7 @@ public void testDescribeConfigFailed() throws PulsarAdminException {
         try {
             describeConfigsResult.all().get();
         } catch (Exception ex) {
+            log.error("Error", ex);
```

Comment:
```suggestion
```

Remove this line because the exception is expected. Or just use `log.info("Error: {}", ex.getMessage()`. BTW, I found here we should add a `fail()` in the `try` block.

Reply:
fixed

Reply:
@BewareMyPower I didn't add "fail()" for this PR, because with "fail()" actually the test will fail.
We can address this problem in the test in a follow up work.
it is not strictly related to this patch